content/blog/2016/2016-11-21-serverless-ci-hyper-sh-jenkins-plugin.adoc

```diff
@@ -0,0 +1,118 @@
+---
+layout: post
+title: "Serverless CI with the new Hyper.sh plugin for Jenkins"
```

Comment:
This is the Jenkins blog, it's safe to assume that all plugins referenced are "for Jenkins" so that's safe to cut.

Also "serverless", while I think is an absurd buzzword, is [generally understood to refer to function-based cloud tooling](https://en.wikipedia.org/wiki/Serverless_computing) like AWS Lambda or Azure Functions. I think it's confusing to use "serverless" here when what's really being conveyed is "Containerized CI".